lib/matplotlib/tests/test_ticker.py

```diff
@@ -590,6 +624,25 @@ def test_values(self, vmin, vmax, expected):
         ticks = sym.tick_values(vmin=vmin, vmax=vmax)
         assert_array_equal(ticks, expected)
 
+    def test_subs(self):
+        fig, ax = plt.subplots()
+        sym = mticker.SymmetricalLogLocator(base=10, linthresh=1, subs=[2.0, 4.0])
+        ax.xaxis.set_major_locator(sym)
+        ax.set_xlim(-10, 10)
```

Comment:
I _think_ you don't need the whole figure?
```suggestion
        sym = mticker.SymmetricalLogLocator(base=10, linthresh=1, subs=[2.0, 4.0])
        sym.create_dummy_axis()
        sym.axis.set_view_interval(-10, 10)
```

Reply:
Should I also modify other tests that could use `create_dummy_axis`?

Reply:
Done